virt/virtual_machines/vm_networking/virt-using-the-default-pod-network-with-virt.adoc

```diff
@@ -1,38 +1,21 @@
 :_content-type: ASSEMBLY
 [id="virt-using-the-default-pod-network-with-virt"]
-= Using the default pod network for virtual machines
+= Configuring the default pod network
 include::_attributes/common-attributes.adoc[]
 :context: virt-using-the-default-pod-network-with-virt
 
 toc::[]
 
-You can use the default pod network with {VirtProductName}. To do so, you must use the `masquerade` binding method. Do not use `masquerade` mode with non-default networks.
-
-xref:../../../virt/virtual_machines/vm_networking/virt-attaching-vm-multiple-networks.adoc#attaching-to-multiple-networks[For secondary networks], use the `bridge` binding method.
+By default, {VirtProductName} is installed with a single, internal pod network. You must configure the network interface of a virtual machine to use the `masquerade` binding mode in order to connect to the `pod` network
```

Comment:
We are not really installed with the pod network. It is already there after OpenShift installation, and our VMs can connect to it. What about:

"Virtual machines in {VirtProductName} can be connected to the internal pod network."

Reply:
Correct. Suggested rewrite: "You can connect a virtual machine to the default internal pod network by configuring its network interface to use the `masquerade` binding mode."

No backticks around "pod". The word is generic in this context.

Reply:
Changed intro to:

You can connect a virtual machine to the default internal pod network by configuring its network interface to use the `masquerade` binding mode